draft-ietf-quic-http.md

```diff
@@ -603,11 +602,10 @@ client making the indicated request.  This trades off network usage against a
 potential latency gain.  HTTP/3 server push is similar to what is described in
 HTTP/2 {{!HTTP2}}, but uses different mechanisms.
 
-Each server push is identified by a unique Push ID. This Push ID is used in a
-single PUSH_PROMISE frame (see {{frame-push-promise}}) which carries the request
-headers, possibly included in one or more DUPLICATE_PUSH frames (see
-{{frame-duplicate-push}}), then included with the push stream which ultimately
-fulfills those promises.
+Each server push is identified by a unique Push ID. This Push ID is used in all
+PUSH_PROMISE frames (see {{frame-push-promise}}) which carries the request
+headers, then included with the push stream which ultimately fulfills those
+promises.
```

Comment:
It might be better to more explicit to say "one or more" than "all", like following, clarifying that the values MUST be the same when sent more than once.

 _This Push ID is used in one or more PUSH_PROMISE frames (see {{frame-push-promise}}) that carry the request headers, then included with the push stream which ultimately fulfills those promises. When the same Push ID is promised on multiple request streams, the decompressed request header fields MUST be byte-for-byte identical._

Reply:
Good call. Done.